spec/features/project_spec.rb

```diff
@@ -146,7 +175,7 @@ def then_i_see_the_content_item_and_its_tag_data
   end
 
   def and_i_click_my_new_projects_name_in_the_project_list
-    within 'table' do
+    within 'table:first-of-type' do
```

Comment:
What does the `first-of-type` and `last-of-type` do?

Reply:
There were added in when two tables were being shown and Capybara didn't know which table to selector. It was a quick solution to use the CSS selectors to find the correct table based on the order in which they show on the page, but this isn't the case anymore because we do not show the table if there is no content (for both projects and the tagging progress tables).

I'll update this selector so it's clearer.